workers/unity/Packages/com.improbable.gdk.core/Components/ISpatialComponentUpdate.cs

```diff
@@ -1,6 +1,13 @@
+using Unity.Entities;
+
 namespace Improbable.Gdk.Core
 {
     public interface ISpatialComponentUpdate
```

Comment:
Do we need both the non-generic and the generic versions of the interface?

Reply:
The non-generic one is there for convenience where we may care that *it is an update* but not *update of what component*